linux/user-and-file-management/file-management-tips-ii/easy-way-of-sharing-files.md

```diff
@@ -19,7 +23,7 @@ File transfer on the network without setting up extra software can be easily ach
 
 `woof` (Web Offer One File) is a script and will run on any machine with *Python* installed.
 
-It can be downloaded at: `www.home.unix-ag.org/ simon/woof.html`.
+You can check download it from the link in the **Learn More** section.
```

Comment:
what do you think about this?

```suggestion
To download it, check the **Learn More** section.
```

Reply:
Uh, what happened. That was not what I wrote haha.

It was supposed to say, `you can check it out at the **Learn More** section.`

Yeah I like that I'll commit it